book/src/keymap.md

```diff
@@ -162,6 +162,7 @@ active language server for the file to work.
 | `a`   | Go to the last accessed/alternate file           | `goto_last_accessed_file`  |
 | `n`   | Go to next buffer                                | `goto_next_buffer`         |
 | `p`   | Go to previous buffer                            | `goto_previous_buffer`     |
+| `;`   | Go to last modification in current file          | `goto_last_modification`   |
```

Comment:
The key should be `g.`, last insertion is `.` (but it triggered because `g.` didn't exist so it fell back onto `.`)

Reply:
Repeat last change is not a regular command, its not in keymap file, and even if i define a g. key combination it still repeats the last insertion.

Reply:
Right, I forgot about that.. I think we need to add some more logic to https://github.com/helix-editor/helix/blob/ebc14d9d206a0960a97fd706bd8fbfbedf2fb203/helix-term/src/ui/editor.rs#L739 So it only triggers if editor.keymaps.pending().is_empty()

Reply:
Might work:
```rust
key!('.') if cxt.editor.keymaps.pending().is_empty() => { 
```